ray-operator/controllers/ray/utils/constant.go

```diff
@@ -154,3 +155,16 @@ const (
 	HeadService    ServiceType = "headService"
 	ServingService ServiceType = "serveService"
 )
+
+// RayOriginatedFromServiceLabelValue generates a RayService value for the label RayOriginatedFromLabelKey
```

Comment:
Will it be better if we unify the two functions `RayOriginatedFromServiceLabelValue` and `RayOriginatedFromJobLabelValue` into a single function?

```go
func FUNC_NAME(crdType CRDType, crName string) string {
  return string(crdType) + "_" + crName
}
```